conda_index/index/__init__.py

```diff
@@ -265,6 +240,16 @@ def _ensure_valid_channel(local_folder, subdir):
             os.makedirs(path)
 
 
+class FileInfo(NamedTuple):
```

Comment:
recently learned about the Python 3.6+ way to do NamedTuple

Reply:
I could use dataclasses. Sometimes we use `NamedTuple` for Python version constraints, which I'm also not worried about here (yet)